__tests__/app.test.js

```diff
@@ -73,4 +92,41 @@ describe('Error handling', () => {
          expect(body).toEqual({"message" : "Invalid path provided - please try again"});
         });
     });
+
+    test('400 status: POST a comment to invalid review datatype should return a message ', () => {
+        return request(app)
+        .post("/api/reviews/bananas/comments")
+        .send({
+            username: "mallionaire",
+            body: "This body shouldn't be here!",
+        })
+        .expect(400)
+        .then(({body})=>{
+            expect(body.message).toEqual("Bad Request")
+        })
+    });
+
+    test('404 status: POST a comment to a review id that doesn\' exist should return a message', () => {
+        return request(app)
+        .post("/api/reviews/999/comments")
+        .send({
+            username: "mallionaire",
+            body: "This body shouldn't be here!",
+        })
+        .expect(404)
+        .then(({body})=>{
+            expect(body.message).toEqual("Review ID Not Found")
+        })
+    });
+
```

Comment:
testing looks good, you could be little more specific with some of the bad request like for example what about a username that isn't in our database, this would be a 404 as we couldn't find the user to add a comment that they made